### PR TITLE
#1232 #1227 #1065 

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

File: src/Common/Core/Impl/Extensions/StringExtensions.cs [hunk 1]
```diff
@@ -104,5 +104,17 @@ public static class StringExtensions {
             }
             return result;
         }
+
+        public static string SurroundWithBackticks(this string s) {
```
Comment (AlexanderSher) [inline src/Common/Core/Impl/Extensions/StringExtensions.cs]: I would've call it `EnsureSurroundedWithBackticks` or something similar

Comment (int19h) [inline src/Common/Core/Impl/Extensions/StringExtensions.cs]: I'd call it `ToRSymbol`.

Comment (int19h) [inline src/Common/Core/Impl/Extensions/StringExtensions.cs]: Or rather `ToRName`, because that's the terminology that R spec uses.

**Comment by int19h** [comment]
:+1:

File: src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs [hunk 2]
```diff
@@ -32,8 +34,10 @@ internal sealed class ShowHelpOnCurrentCommand : PackageCommand {
         private readonly IRInteractiveWorkflow _workflow;
         private readonly IActiveWpfTextViewTracker _textViewTracker;
 
+        private IActiveRInteractiveWindowTracker _activeReplTracker;
         public ShowHelpOnCurrentCommand(IRInteractiveWorkflow workflow, IActiveWpfTextViewTracker textViewTracker) :
             base(RGuidList.RCmdSetGuid, RPackageCommandId.icmdHelpOnCurrent) {
+            _activeReplTracker = VsAppShell.Current.ExportProvider.GetExportedValue<IActiveRInteractiveWindowTracker>();
```
Comment (AlexanderSher) [inline src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs]: please provide it as ctor parameter so command can be tested without stub injection into VsTestCompositionCatalog

File: src/Common/Core/Impl/Extensions/StringExtensions.cs (hunk 1 above)
Comment (AlexanderSher) [inline src/Common/Core/Impl/Extensions/StringExtensions.cs]: Do we have any R-specific code in Common?

Comment (int19h) [inline src/Common/Core/Impl/Extensions/StringExtensions.cs]: You're right, we don't. So it should be moved to where `ToRStringLiteral` is.

**Pull request merged**